src/field/FieldParamsPointers.hh

```diff
@@ -0,0 +1,77 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file FieldParamsPointers.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/Macros.hh"
+#include "base/Types.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Host and device parameters for propagation in a magnetic field
+ */
+struct FieldParamsPointers
+{
+    //! accuracy for integration of one step = 0.01 mm
+    //    real_type delta_one_step;
+
+    //! the closest miss distrance = 0.25 mm
+    real_type delta_chord;
+
+    //! accuracy of intersection of a boundary crossing  = 0.0001 mm
+    real_type delta_intersection;
+
+    //! the relative error = 1.0e-5
+    real_type epsilon_step;
+
+    //! min step size = 1.0e-5
+    real_type minimun_step;
```

Comment:
Typo in this variable name -- is this variable used, or is the typo elsewhere in the code?

Reply:
The typo is fixed in all places where it is actually set or used.